nanome_rmsd/rmsd_menu.py

```diff
@@ -384,18 +384,29 @@ def align_box_button_pressed_callback(button):
             self._plugin.update_content(align_box_button)
             self._plugin.update_content(align_box_text)
 
+        # global <=> local
+        def global_local_button_pressed_callback(button):
+            if self._plugin.args.select == "global":
+                self.update_args("select","local")
+                global_local_button.text.value.set_all("Local")
+            else:
+                self.update_args("select","global")
+                global_local_button.text.value.set_all("Global")
+            self._plugin.update_content(global_local_button)
+            self._plugin.update_content(global_local_text)
```

Comment:
Doesn't look like the "global_local_text" object changed at all. Do we need to update it here?